tests/test_lsst.py

```diff
@@ -374,7 +374,7 @@ def test_pupil_coords_from_pixel_coords(self):
         chip_name_possibilities = ('R:0,1 S:1,1', 'R:0,3 S:0,2', 'R:4,2 S:2,2', 'R:3,4 S:0,2')
 
         chip_name_list = [chip_name_possibilities[ii]
-                          for ii in rng.random_integers(0,3,n_samples)]
+                          for ii in rng.randint(0,3,n_samples)]
```

Comment:
Looks like you're right.  I don't think it really matters in this case, though.  All I am doing is drawing a random sample of chip names from `chip_name_possibilities`.  The code now ignores the last entry in `chip_name_possibilities`, which doesn't really affect the usefulness of the test.